### ISSUE TITLE
Upgrade 62.2 => 63.3

### ISSUE THREAD

**Issue by nachocifu** [react-native-community/upgrade-support#113]
## Environment
```
System:
    OS: macOS 10.15.6
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
    Memory: 5.11 GB / 32.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.18.2 - /usr/local/opt/node@12/bin/node
    npm: 6.14.5 - /usr/local/opt/node@12/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 14.0, DriverKit 19.0, macOS 10.15, tvOS 14.0, watchOS 7.0
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5977832
    Xcode: 12.0/12A7209 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.11.0 => 16.11.0
    react-native: 0.62.2 => 0.62.2
  npmGlobalPackages:
    react-native-rename: 2.4.1
    react-native: 0.61.5
```

## Upgrading version

`0.63.3`

## Description
Are there any steps on how to upgrade iOS `ios/RnDiffApp.xcodeproj/project.pbxproj` file? Using `https://react-native-community.github.io/upgrade-helper/?from=0.62.2&to=0.63.3` is kind of a nightmare.

**Comment by tristanbrandt014** [comment]
What worked for me was:

```bash
# In your project root
yarn
cd ios

# I had an issue with pod install as well, but this sorted it out. 
# Ignore if you can pod install without a problem
pod update Folly --no-repo-update

pod install
```

Then in `Xcode > YourApp > Build Phases > Copy Bundle Resource`, delete `LaunchScreen.xib`

In the same section, click the add button, and `Add Other...`. Select `LaunchScreen.storyboard` (Should be under ios/YourApp after you've done the upgrade). Ensure `Copy items if needed` is selected, and hit finish.

The app should build after this. You can edit `LaunchScreen.storyboard` with Xcode if necessary.

**Comment by avencat** [comment]
Thanks @tristanbrandt014 the `pod update Folly --no-repo-update` fixed my issue! 🎉

**Comment by wilkuintheair** [comment]
Updating Folly command could be added to the docs, I've had this issue probably every time I updated RN projects to 0.63.3.
If anyone needs some context, here's the error I had when `pod install` after upgrade:
```
[!] CocoaPods could not find compatible versions for pod "Folly":
  In snapshot (Podfile.lock):
    Folly (from `../node_modules/react-native/third-party-podspecs/Folly.podspec`)

  In Podfile:
    Folly (from `../node_modules/react-native/third-party-podspecs/Folly.podspec`)

    React-Core (from `../node_modules/react-native/`) was resolved to 0.63.3, which depends on
      Folly (= 2020.01.13.00)
```

**Comment by harshitsmt** [comment]
Hi Can anyone help me with this 
https://react-native-community.github.io/upgrade-helper/?from=0.61.5&to=0.63.3

Only the file ios/RnDiffApp.xcodeproj/project.pbxproj left for modification how can I do it?

**Comment by nachocifu** [comment]
@harshitsmt id try following [my steps](https://github.com/react-native-community/upgrade-support/issues/113#issuecomment-712390357). 
Im gonna close this issue as I believe its resolved for 62.2 => 63.3 specific scenario. Feel free to open another for your specific case.
